pkg/cloud/services/eks/cluster.go

```diff
@@ -867,3 +908,10 @@ func clusterUpdatingStateRetryable(ctx context.Context, input *eks.DescribeClust
 	}
 	return true, nil
 }
+
+func convertUpgradePolicy(input ekscontrolplanev1.UpgradePolicy) ekstypes.SupportType {
```

Comment:
We sometimes but these in the "convert" package when converting  to/from SDK types.